### PR TITLE
chore(scalasdk): rename scala-value-entity-customer-registry

### PR DESCRIPTION
**Pull request by RayRoestenburg**
to scala-valueentity-customer-registry (consistency with other project directories).

### CODE REVIEW

**Review by raboof — APPROVED** [review]
Hmm, I guess that seems reasonable - would that mean we'd also want to rename the maven archetypes?

**Comment by RayRoestenburg** [comment]
@raboof yeah, sounds like a good cleanup.

File: samples/scala-valueentity-customer-registry/scala-value-entity-customer-registry.sublime-project [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (patriknw) [inline samples/scala-valueentity-customer-registry/scala-value-entity-customer-registry.sublime-project:1]: could you remove all sublime files and add to gitignore

**Review by patriknw — COMMENTED** [review]
good to rename, some sublime files...

File: samples/scala-valueentity-customer-registry/scala-value-entity-customer-registry.sublime-project (hunk 1 above)
Comment (RayRoestenburg) [reply samples/scala-valueentity-customer-registry/scala-value-entity-customer-registry.sublime-project:1]: ah crap, will do.

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**